run.sh

```diff
@@ -53,9 +48,11 @@ sh ./kind-registry.sh
 # Build and deploy the server image
 echo "Building polaris image..."
 ./gradlew \
-  :polaris-server:build \
+  :polaris-server:assemble \
   :polaris-server:quarkusAppPartsBuild --rerun \
-  $ECLIPSE_LINK_DEPS \
+  :polaris-admin:assemble \
+  :polaris-admin:quarkusAppPartsBuild --rerun \
+  -Dquarkus.container-image.tag=postgres-latest \
```

Comment:
I think we don't need the special tag anymore. The default image is built with the Postgres JDBC driver afaict.